data/migrateDistributions.js

```diff
@@ -0,0 +1,169 @@
+import { updateFieldsInCollection } from './updateFieldsInCollection.js';
+import {
+  addPlanDistributions,
+  initDistributions,
+} from '../routes/distributionMethods.js';
+import * as db from './db.js';
+import Courses from '../model/Course.js';
+import SISCV from '../model/SISCourseV.js';
+import Plans from '../model/Plan.js';
+import Years from '../model/Year.js';
+import Distributions from '../model/Distribution.js';
+import FineRequirements from '../model/FineRequirement.js';
+import mongoose from 'mongoose';
+
+mongoose.set('strictQuery', false);
+// updateModels();
+// setDepartmentInCourses();
+// setTagInCourses();
+// addDistributions();
+
+// NOTE: FineRequirements and Distributions (and Majors) are collections with 0 documents
+// They should not need any updating.
+
+// UPDATE fields on models
+async function updateModels() {
+  try {
+    // update course schema
+    await updateFieldsInCollection(
+      Courses,
+      { area: { $exists: true } },
+      { $rename: { area: 'areas' } },
+    );
+    await updateFieldsInCollection(Courses, {}, { distribution_ids: [] });
+    await updateFieldsInCollection(Courses, {}, { fineReq_ids: [] });
+    // // update plan schema
+    await updateFieldsInCollection(
+      Plans,
+      {},
+      { $rename: { majors: 'major_ids' } },
+    );
+    await updateFieldsInCollection(
+      Plans,
+      {},
+      { $unset: { distribution_ids: '' } },
+    );
+    // // update year schema
+    await db.connect();
+    const years = await Years.find({}).exec();
+    console.log(years.length, ' year documents matched.');
+    let count = 0;
+    for (let year of years) {
+      console.log(year.plan_id, Array.isArray(year.plan_id));
+      if (year.plan_id && Array.isArray(year.plan_id)) {
+        console.log(year.plan_id[0]);
+        year.plan_id = year.plan_id[0];
+        await year.save();
+        count++;
+      }
+    }
+    console.log(count, ' year documents modified.');
+  } catch (err) {
+    console.log(err);
+  }
+}
+
+async function setDepartmentInCourses() {
```

Comment:
don't remember why this was needed but seems to have been an issue before so will run for good measure. 

basically sets course department field by looking up course in SISCourseV